dnsseed.go

```diff
@@ -198,20 +198,36 @@ func main() {
 	}
 
 	if len(cfg.Seeder) != 0 {
-		ip := net.ParseIP(cfg.Seeder)
+		// Prepare the seeder address, supporting either a simple IP with default network port
+		// or a full IP:port format
+		seederIp := cfg.Seeder
+		seederPort := peersDefaultPort
+
+		addressPart := strings.Split(cfg.Seeder, ":")
```

Comment:
You can use `network.NormalizeAddress`

Reply:
I dont' see how to use this here.

We need to support the case of absence of user-specified seeder, in which case the seeder ip is resolved by `net.LookupHost(seederIp)`. And ultimately, the point is to build an `appmessage.NetAddress`, providing ip and port separately.